dms/models/directory.py

```diff
@@ -265,11 +265,15 @@ def _get_own_root_directories(self, user_id):
     def check_access(self, operation, raise_exception=False):
         res = super(DmsDirectory, self).check_access(operation, raise_exception)
         if self.env.user.has_group("base.group_portal"):
-            if self.id in self._get_ids_without_access_groups(operation):
+            if self.ids in self._get_ids_without_access_groups(operation):
                 res = False
         # Fix show breadcrumb with share button (public)
         if self.env.user.has_group("base.group_public"):
             res = True
+        # Fix record_ref_mail_follower_ids
```

Comment:
Possibly you'll have to remove references to followers, as it's not the main point here.